examples/echo-arg/main_test.go

```diff
@@ -63,7 +63,7 @@ func TestMainMethod(t *testing.T) {
 	}
 	for _, tt := range tests {
 		fmt.Println(tt.name)
-		output, exitCode, err := collector.RunBinary(binPath, "TestBincoverRunMain", []string{}, tt.args)
+		output, exitCode, err := collector.RunBinary(binPath, "TestBincoverRunMain", []string{}, tt.args, "")
```

Comment:
you probably want an io.Reader instead of a string. That's more flexible and is the pattern the go stdlib follows

Reply:
good call